recipes/en_core_web_sm/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+size:
+  - sm
+  - md
+  - lg
```

Comment:
You don't need multiple outputs and this build matrix. IDK the tradeoffs between the two, but we should only be using one mechanism I think.